src/test/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJobTest.java

```diff
@@ -65,6 +66,9 @@
 public class AnnotationJobTest {
     private static final String MOCK_VEP = "/mockvep.pl";
     private static final String MONGO_DUMP = "/dump/VariantStatsConfigurationTest_vl";
+    private static final String OUTPUT_DIR_ANNOTATION = "/tmp/";
```

Comment:
I think that this can only be done when all the annotation steps are using the jobParameters. If we mix the rule and properties then the files might end up in different folders. The rule for example is used in VepAnnotationGeneratorStepTest because is self-contained.